UI/window-basic-main.cpp

```diff
@@ -6359,7 +6350,7 @@ static void RenameListItem(OBSBasic *parent, QListWidget *listWidget,
 }
 
 void OBSBasic::SceneNameEdited(QWidget *editor,
-			       QAbstractItemDelegate::EndEditHint endHint)
+			       QAbstractItemDelegate::EndEditHint)
```

Comment:
Parameter can be removed

Reply:
Needs to be a auto generated slot.

Reply:
See https://github.com/obsproject/obs-studio/pull/8167#discussion_r1118067680

Reply:
Fixed